3rdparty/ndsrvp/include/core.hpp

```diff
@@ -0,0 +1,520 @@
+#ifndef OPENCV_NDSRVP_CORE_HPP_INCLUDED
```

Comment:
Please add license header to all source files (https://github.com/opencv/opencv/wiki/Coding_Style_Guide#file-structure). You can also add respective copyrights afterwards.